public/src/components/channelManagement/epicTests/epicTestsForm.tsx

```diff
@@ -1,214 +1,56 @@
-import React from 'react';
-import EpicTestEditor from './epicTestEditor';
-import { Region } from '../../../utils/models';
-
+import { FrontendSettingsType } from '../../../utils/requests';
+import { TestsForm } from '../testsForm';
+import { getDefaultTest, getDefaultVariant } from './utils/defaults';
 import {
   AMP_EPIC_CONFIG,
   APPLE_NEWS_EPIC_CONFIG,
   ARTICLE_EPIC_CONFIG,
   ARTICLE_EPIC_HOLDBACK_CONFIG,
-  ArticlesViewedSettings,
-  ContributionFrequency,
-  Cta,
-  DeviceType,
   EpicEditorConfig,
-  Image,
   LIVEBLOG_EPIC_CONFIG,
-  SecondaryCta,
-  Status,
-  Test,
-  UserCohort,
-  Variant,
 } from '../helpers/shared';
-import TestsForm, { InnerComponentProps } from '../testEditor';
-import TestsFormLayout from '../testsFormLayout';
-import Sidebar from '../sidebar';
-import { FrontendSettingsType } from '../../../utils/requests';
-import { getDefaultTest, getDefaultVariant } from './utils/defaults';
-import { ControlProportionSettings } from '../helpers/controlProportionSettings';
-
-export enum TickerEndType {
```

Comment:
all of these models have moved to `/models/epic.ts`, in common with the banner/header models

Reply:
This refactoring is really welcome - less having to search through the codebase for obscure defs == less tech debt.